RUFAS/output_manager.py

```diff
@@ -2200,3 +2201,396 @@ def run_startup_sequence(
+            "function": self.validate_graph_detail_options.__name__,
+        }
+        key_validators: dict[str, Callable[[Any, str], None]] = {
+            "type": self.validate_graph_type,
+            "filters": self.validate_list_of_strings,
+            "variables": self.validate_list_of_strings,
+            "filter_by_exclusion": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "customization_details": self.validate_customization_details,
+            "legend": self.validate_list_of_strings,
+            "display_units": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "omit_legend_prefix": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "omit_legend_suffix": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "expand_data": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "use_fill_value_in_gaps": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "use_fill_value_at_end": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "mask_values": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "use_calendar_dates": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "data_significant_digits": partial(self.validate_type, expected=int, type_label="an integer"),
+            "title": partial(self.validate_type, expected=str, type_label="a string"),
+            "slice_start": partial(self.validate_type, expected=int, type_label="an integer"),
+            "slice_end": partial(self.validate_type, expected=int, type_label="an integer"),
+        }
+        for key, value in details.items():
+            if key == "date_format":
+                Utility.validate_date_format(value)
+                continue
+            if key == "fill_value":
+                continue
+            if key not in key_validators:
+                self.add_error(
+                    "Unknown key in graph details.",
+                    f"Key: {key}, is not a supported option in graph details.",
+                    info_map,
+                )
+                continue
+            key_validators[key](value, key)
+
+    def validate_aggregator(self, value: Any, content_name: str) -> None:
+        """
+        Validate the aggregator option provided.
+
+        Parameters
+        ----------
+        value : Any
+            The aggregator option to validate.
+        content_name : str
+            The corresponding filter option to provide in error reporting.
+
+        Returns
+        -------
+        None
+
+        """
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self.validate_aggregator.__name__,
+        }
+        self.validate_type(value, content_name, expected=str, type_label="a string")
+
+        allowed = {
```

Comment:
```suggestion
        supported = {
```
I feel like `allowed` seems like the wrong word here. Maybe `supported_aggregations` or just `supported`?